### PR TITLE
Fix/split button quick fixes

### PR DESCRIPTION
**Pull request by jackkav**
Closes INS-1652

### CODE REVIEW

File: packages/insomnia/src/ui/css/components/request-url-bar.less [hunk 1]
```diff
@@ -58,6 +54,10 @@
   & > .dropdown {
     height: 100%;
   }
+  .urlbar__send-context,
+  .surprise {
+    border-left: 1px solid var(--hl);
```
Comment (gatzjames) [inline packages/insomnia/src/ui/css/components/request-url-bar.less]: Should we use the same color as the other borders in the app?
```suggestion
border-left: 1px solid var(--hl-md);
```

**Comment by gatzjames** [comment]
Tested and works nice!

File: packages/insomnia/src/ui/css/components/request-url-bar.less (hunk 1 above)
Comment (jackkav) [inline packages/insomnia/src/ui/css/components/request-url-bar.less]:
```suggestion
    border-left: 1px solid var(--hl-md);
```

**Review by gatzjames — APPROVED** [review]
🚀

**Pull request merged**